docs/usage/examples/self-hosting.md

```diff
@@ -396,3 +396,35 @@ The logging level output is controlled by the Bunyan logging library.
 |    40 | warn    |
 |    50 | error   |
 |    60 | fatal   |
+
+## Self-signed TLS/SSL certificates
+
+Renovate and invoked helper programs (e.g. Git, npm) use a secure TLS connection (e.g. HTTPS) to connect to source code platforms and dependency hosts.
+If the systems use any self-signed certificates or certificate authorities then Renovate needs to be configured to trust these additional certificates.
+
+For the main Renovate Node.js application set the environment variable [`NODE_EXTRA_CA_CERTS=/usr/local/share/ca-certificates/self-signed-certificate.crt`](https://nodejs.org/api/cli.html#node_extra_ca_certsfile).
+This ensures that the Renovate application itself trusts the `self-signed-certificate.crt` and can establish secure connections to systems using that certificate or certificates signed by this certificate authority.
+
+The helper programs (e.g. Git, npm) use the system trust store.
+For them to trust a self-signed certificate you must add it to the systems trust store.
+On Ubuntu/Debian and many Linux-based systems, this can be achieved by copying the self-signed certificate (e.g. `self-signed-certificate.crt`) to `/usr/local/share/ca-certificates/` and running [`update-ca-certificates`](https://manpages.ubuntu.com/manpages/xenial/man8/update-ca-certificates.8.html) to update the system trust store afterwards.
+
+If you are using the official [Renovate Docker image](#docker) then the best way is to build your own Docker image with the self-signed certificate added to the standard Renovate Docker image.
```

Comment:
```suggestion
If you're using the official [Renovate Docker image](#docker) then we recommend you follow these steps:

 1. Build your own Docker image
 1. Add the self-signed certificate to the standard Renovate Docker image
```

How about making a list of the two things you need to do?

Reply:
I would argue that this is indeed onestep of building your own docker image with the self-signed certificates. 
Maybe this is better:
```suggestion
If you're using the official [Renovate Docker image](#docker) then we recommend you to add the self-signed certificate to it and build your own modified Docker image.
```

Reply:
I like your suggestion, but you should remove the word `to` before you apply it. 😉